next.config.js

```diff
@@ -8,6 +8,7 @@ const nextConfig = {
     },
     images: {
         domains: ['images.ctfassets.net'],
+        unoptimized: true,
```

Comment:
What is this for?

Reply:
An error appeared in the console related to export, so it offered me 2 options either to delete the export or to insert this line of code.